hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceConfigurationBuilder.java

```diff
@@ -505,7 +537,7 @@ private static void determineMetricSetPrometheus(ModelNode config,
 
                     MetricType<PlatformNodeLocation> totalSpace = new MetricType<PlatformNodeLocation>(null,
                             Constants.FILE_STORE_TOTAL_SPACE,
-                            new AttributeLocation<PlatformNodeLocation>(
+                            new AttributeLocation<>(
```

Comment:
I like using <> when allowed - makes the code cleaner. I'm fine with those changes to remove the generic type inside <>